src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/MS/Internal/MarkupCompiler/VersionHelper.cs

```diff
@@ -0,0 +1,215 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Modified from https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/VersionHelper.cs
```

Comment:
Refer to the sha that formed the source (instead of blob/master/... which is ever changing)

Reply:
Maybe we should use commit instead of master strings